src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -261,6 +269,9 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 	}, [fieldDef?.disabled, options]);
 
 	const deleteSelectedOption = (newOptions: MosaicLabelValue[]) => {
+		if (newOptions.length <= fieldDef.inputSettings?.selectLimit) {

```

Comment:
I think the condition should only be "less than" because if we keep the "equal" we would run into an issue with the code on line 68 `value.length >= fieldDef.inputSettings?.selectLimit ? setCheckboxListDisabled(true)` because we're saying: "When the amount of options is equal than or bigger than the limit then we disable the list" but we're also saying "When the amount of options is equal than or less than the limit then we enable the list".

```suggestion
		if (newOptions.length < fieldDef.inputSettings?.selectLimit) {

```